tokenizers/src/models/unigram/trainer.rs

```diff
@@ -421,6 +424,11 @@ impl UnigramTrainer {
             Vec::with_capacity(self.vocab_size.try_into().unwrap());
         // XXX: Make sure unk exists and are ids 0
```

Comment:
If we want to get rid of unk_token in the general case, I think we need to remove it from here no ? Or are we still keeping it for now, because we can't get rid of it in the trainer ?

Reply:
We are keeping it for now. It leads to breaking changes that are best left for another PR, for another release.